src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private User user = new User();
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @Before
+    public void setUser() {
+        user.setAge(19);
+        user.setLogin("Genareb");
+        user.setPassword("asdfgh");
+    }
+
+    @Test
+    void register_nullUser_notOk() throws RuntimeException {
+        try {
+            registrationService.register(null);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("RuntimeException should be thrown in user == null case");
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        setUser();

```

Comment:
```suggestion
```
There is an @BeforeEach annotation instead of @Before. Then you don't have to do this everywhere